Raven/RavenClient.swift

```diff
@@ -400,7 +383,7 @@ public class RavenClient : NSObject, NSURLConnectionDelegate, NSURLConnectionDat
             }
         }
 
-        #if os(iOS)
+        #if os(iOS) || os(tvOS)
             if (tags["OS version"] == nil) {
```

Comment:
it's a different os but not a big deal, most of the people probably log on a different sentry project ios and apple tv